src/System.Net.Requests/tests/HttpWebRequestTest.netstandard1.7.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Yes for any projects that need to build in multiple configurations then we have been primarily splitting into other files and including the configuration as part of the file name. See https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md#code-file-naming-conventions.